demo.tape

```diff
@@ -0,0 +1,64 @@
+# Demo of Geodesic
+
+Output docs/demo.gif
+
+Set Theme "Monokai Vivid"
+
+#Set FontFamily "FiraCode Nerd Font"
+Set FontFamily "Hack Nerd Font"
+Set FontSize 14
+
+Set TypingSpeed 40ms
+
+Set Framerate 30
+
+Set WindowBar Colorful
+
+Set CursorBlink true
+
+Set BorderRadius 8
+
+Set Margin 20
+Set MarginFill "#674EFF"
+
+Set Padding 12
+
+Set Width 1200
+Set Height 600
+
+Set PlaybackSpeed 1
+
+Set Shell bash
+
+Type "# After having run `make all`, simply run `geodesic` to enter the interactive shell" Enter
```

Comment:
IMO make run is more commonly used, as people may customize the startup script name, but make run is always the same

```suggestion
Type "# After having run `make all`, simply run `make run` (or `geodesic`) to enter the interactive shell" Enter
```